.env.example

```diff
@@ -1,5 +1,6 @@
 DATABASE_URL=postgres://pguser:pgpassword@localhost:5432/bms
-RABBITMQ_ENDPOINT=127.0.0.1
+RABBITMQ_SSL_ENABLED=false
+RABBITMQ_ENDPOINT=http://127.0.0.1
```

Comment:
Maybe we can deduce this from the protocol in the connection string? `amqps://` would be ssl enabled, `amqp://` would be ssl disabled?

Just a suggestion, feel free to merge either way if you disagree.

Reply:
fair enough, one env less